met-web/src/components/engagement/view/SurveyBlock.tsx

```diff
@@ -58,7 +58,8 @@ const SurveyBlock = ({ startSurvey }: SurveyBlockProps) => {
                                 data-testid="SurveyBlock/view-feedback-button"
                                 disabled={!surveyId}
                                 onClick={() => {
-                                    navigate(`/engagements/${savedEngagement.id}/dashboard`);
+                                    navigate(`/engagements/${savedEngagement.id}/dashboard/1`);
+                                    window.scrollTo(0, 0);
```

Comment:
This would only help for this instance. Let's try updating it at the route level like what we're doing for the AuthenticatedRoutes 
![image](https://github.com/bcgov/met-public/assets/91914654/eab51dfa-9a6d-40d9-9f1c-5e9b39292fcf)

Reply:
fixed good idea